bcManager/bcManager.py

```diff
@@ -1969,6 +2014,12 @@ def get_latest_player_data_by_platform_name(self, guild, platform, plat_name):
                         return player
         return {}
 
+    async def iter_gather(result):
```

Comment:
this is unused

Reply:
It's something I want for later as a quick async generator helper function. There is more work to be done here and I see myself using it in the future and for testing purposes.